create-quasar/templates/app/quasar-v2/ts-webpack/index.js

```diff
@@ -7,7 +7,8 @@ module.exports = async function ({ scope, utils }) {
       message: 'Pick a Vue component style:',
       initial: 0,
       choices: [
-        { title: 'Composition API', value: 'composition', description: 'recommended' },
+        { title: 'Composition API with <script setup>', value: 'composition-setup', description: 'recommended' },
+        { title: 'Composition API', value: 'composition' },
```

Comment:
As `<script setup>` is not yet widely used in our docs, let's make it the second option. But, let's add `recommended` to both. So, people who know <script setup> will not get confused about why it's not recommended, and those who don't know it continues with the non-script setup which is widely used in the docs, examples, etc. and they don't get stuck. When we update the docs to address that, we might consider switching the order or the recommendations.

```suggestion
        { title: 'Composition API', value: 'composition', description: 'recommended' },
        { title: 'Composition API with <script setup>', value: 'composition-setup', description: 'recommended' },
```